erpc_c/port/erpc_threading_zephyr.cpp

```diff
@@ -53,14 +50,12 @@ void Thread::init(thread_entry_t entry, uint32_t priority, uint32_t stackSize, t
 void Thread::start(void *arg)
 {
     m_arg = arg;
-
-    erpc_assert(m_stack && "Set stack address");
-    k_thread_create(&m_thread, m_stack, m_stackSize, threadEntryPointStub, this, NULL, NULL, m_priority, 0, K_NO_WAIT);
+    m_thread_id = k_thread_create(&m_thread, saas_thread_stack, 4096, (k_thread_entry_t) threadEntryPointStub, this, NULL, NULL, m_priority, 0, K_NO_WAIT);
```

Comment:
Minor nit: Please make the stack size a #define so the usage here and above in K_THREAD_STACK_DEFINE can be changed in just a single place.  You just *know* someone is gonna change it one place and forget the other.